cxl/memdev.c

```diff
@@ -4373,12 +4378,13 @@ static int action_cmd_get_cxl_membridge_stats(struct cxl_memdev *memdev,
 {
 	if (cxl_memdev_is_active(memdev)) {
 		fprintf(stderr, "%s: memdev active, abort cxl membridge stats\n",
-    			cxl_memdev_get_devname(memdev));
+                cxl_memdev_get_devname(memdev));
```

Comment:
git commit was giving space related error here line 4381 and 4236, and hence you would see space / tab related changes but those changes are not related to code

Reply:
not sure why these are giving errors.